react-native-libraries.json

```
@@ -6927,14 +6927,17 @@
     "android": true
   },
   {
-    "githubUrl": "https://github.com/realm/realm-js",
+    "githubUrl": "https://github.com/realm/realm-js/tree/main/packages/realm",
     "npmPkg": "realm",
     "examples": [
-      "https://github.com/realm/realm-js/tree/master/examples/ReactExample",
-      "https://github.com/mongodb-university/realm-tutorial-react-native"
+      "https://github.com/realm/realm-js/tree/main/examples/rn-todo-list",
+      "https://github.com/realm/realm-js/tree/main/examples/rn-multiple-realms",
+      "https://github.com/realm/realm-js/tree/main/examples/rn-connection-and-error",
+      "https://www.mongodb.com/docs/atlas/app-services/tutorial/react-native/"
     ],
     "ios": true,
-    "android": true
+    "android": true,
+    "expo": true
```

Hello @elle-j, thank you for the updates! 👍

```suggestion
    "android": true
```

The `expo` flag signifies the support Expo Go client, and since the library uses native code, it won't be supported (until pre-bundled). Every RN package can work with Expo when using Development Client, and it looks like it's the way recommended in template:
* https://github.com/realm/realm-js/tree/main/templates/expo-template#readme

---

Thanks for the clarification! Wasn't entirely clear in the README that it referred to Expo Go. I've pushed a commit removing the `expo` entry 👍

---

We will update the description, thanks for the feedback! 👌